extensions/wikia/VisitSource/scripts/VisitSource.js

```diff
@@ -0,0 +1,9 @@
+$(function() {
+	'use strict';
+
+	var sessionVisitSource = new VisitSource('WikiaSessionSource', window.wgCookieDomain),
+		lifetimeVisitSource = new VisitSource('WikiaLifetimeSource', window.wgCookieDomain, false);
```

Comment:
The last argument is not too descriptive itself. How about moving it to a var on the top of the closure, named like "isSessionOrigin"?